server/controllers/CommentController.js

```diff
@@ -0,0 +1,177 @@
+import db from '../models';
+import response from '../helpers/response';
+import TokenManager from '../helpers/TokenManager';
+
+
+const { Article, Comment } = db;
+
+/**
+ * @class UserController
+ */
+class CommentController {
+  /**
+   *
+   * @description Allow user to comment on an article
+   * @static
+   * @param {*} req Express Request object
+   * @param {*} res Express Response object
+   * @returns {object} Json response
+   * @memberof CommentController
+   */
+  static async addComment(req, res) {
+    try {
+      const token = req.headers.authorization.split(' ')[1];
+      const { userId } = TokenManager.verify(token);
```

Comment:
Why are you getting the header token again since the `checkIfUserIsAuthenticated` middleware already decoded the token and passes the current `user` object along with the `req` object

Reply:
Thanks have fixed it